here-naksha-app-service/src/main/java/com/here/naksha/app/service/http/tasks/SpaceApiTask.java

```diff
@@ -99,6 +101,14 @@ protected void init() {}
     }
   }
 
+  private XyzResponse executeDeleteSpace() {
+    final String spaceId = extractMandatoryPathParam(routingContext, SPACE_ID);
+    final WriteXyzFeatures wr = new WriteXyzFeatures(SPACES).delete(new Space(spaceId));
```

Comment:
Unfortunately we need the whole object - that's because we need to know which `collectionId` is defined on the Space, so the flow from API perspective looks like that:
1) get `spaceId` from URI
2) fetch Space with given `spaceId` from storage
3) forward this Space (along with its `collectionId` property) to proper handlers

Reply:
I still don't think we require entire Space object here. May be there is difference in understanding which we can discuss on call, after I have looked at all other comments.

Reply:
We won't know the collectionId having just the spaceId - hence, we need to obtain the feature from db before

Reply:
I will move SpaceId/CollectionId shift to the handler - as discussed on call